### PR TITLE
Fixes #14124 - add auto complete to content host page

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/hosts/host.factory.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ angular.module('Bastion.hosts').factory('Host',
     ['BastionResource', function (BastionResource) {
         var resource = BastionResource('/api/v2/hosts/:id/:action', {id: '@id'}, {
             update: {method: 'PUT'},
-            updateHostCollections: {method: 'PUT', params: {action: 'host_collections'}}
+            updateHostCollections: {method: 'PUT', params: {action: 'host_collections'}},
+            autocomplete: {method: 'GET',isArray: true, params: {id: 'auto_complete_search'}}
```
Comment (thomasmckay) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/hosts/host.factory.js]: missing space after comma

File: test/controllers/api/v2/host_autocomplete_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# encoding: utf-8
+
+require "katello_test_helper"
+
+module Katello
+  class Api::V2::HostPackagesControllerTest < ActionController::TestCase
```
Comment (daviddavis) [inline test/controllers/api/v2/host_autocomplete_controller_test.rb]: HostPackagesControllerTest?

Comment (jlsherrill) [inline test/controllers/api/v2/host_autocomplete_controller_test.rb]: whoops

**Comment by thomasmckay** [comment]
ACK

**Pull request merged**